src/main/java/seedu/address/commons/core/SavedFilePath.java

```diff
@@ -0,0 +1,67 @@
+package seedu.address.commons.core;
+
+import static seedu.address.commons.util.FileUtil.isFileExists;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Objects;
+import java.util.logging.Logger;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ *  A Serializable class that contains the saved file path of the user.
+ */
+public class SavedFilePath {
+    private static final String DEFAULT_FILE_PATH = System.getProperty("user.dir");
+    private static final Logger logger = LogsCenter.getLogger(SavedFilePath.class);
+
+    private String value;
+
+    /**
+     * Constructs a {@code SavedFilePath} with the default value.
+     */
+    public SavedFilePath() {
+        this.value = DEFAULT_FILE_PATH;
+    }
+
+    /**
+     * Constructs a {@code SavedFilePath} with the given value.
+     */
+    @JsonCreator
+    public SavedFilePath(@JsonProperty("value") String value) {
+        Path path = Paths.get(value);
+        if (isFileExists(path)) {
+            this.value = path.toAbsolutePath().toString();
```

Comment:
Absolute path is already ensured in TagCommand, but its still okay to get absolute path again here I think.

Reply:
I dont think this is related to TagCommand, since the value in here is from what being read from the json file. The absolutePath in here is to ensure it is a absolute path if the user try to manipulate the data inside the json file to use relative path.

Reply:
I get it now.